packages/io/stl-serializer/CSGToStla.js

```diff
@@ -1,50 +1,46 @@
-const { ensureManifoldness } = require('@jscad/io-utils')
-const {isCSG} = require('@jscad/csg')
+const { geometry } = require('@jscad/modeling')
 
-// objects must be an array of CSG objects (with polygons)
+// objects must be an array of 3D geomertries (with polygons)
```

Comment:
Small note: perhaps in the future we should specify that they are of type Geom3 (or any similar things)